### PR TITLE
Attempt to add date_format support

### PR DESCRIPTION
**Pull request by xinthose**
Hello.  This is my attempt to add date_format support to the client and event connected/time columns.  I have not tested the code.  I am not sure how to expose `servers.date_format` in `Clients.vue` and `Events.vue` `formatTime` and `eventTime` functions.

### CODE REVIEW

**Comment by AuspeXeu** [comment]
Thanks for the PR. Could you only commit the files that actually need changing? At the moment there is a lot going on in this PR. :)

**Comment by xinthose** [comment]
OK.  Let me try again.

**Comment by AuspeXeu** [comment]
You can also edit PRs by updating the formed branch :)
